cmd/influxd/launcher/cmd.go

```diff
@@ -182,11 +186,13 @@ func newOpts(viper *viper.Viper) *InfluxdOpts {
 		BoltPath:   filepath.Join(dir, bolt.DefaultFilename),
 		EnginePath: filepath.Join(dir, "engine"),
 
-		HttpBindAddress:      ":8086",
-		HttpTLSMinVersion:    "1.2",
-		HttpTLSStrictCiphers: false,
-		SessionLength:        60, // 60 minutes
-		SessionRenewDisabled: false,
+		HttpBindAddress:       ":8086",
+		HttpReadHeaderTimeout: 10 * time.Second,
```

Comment:
Why not set defaults for `HttpReadTimeout` and `HttpWriteTimeout` as well? Presumably these being set to have no timeout presents similar risks to resource starvation

Reply:
Checking with the compute team from Cloud to see if they set any values for these which we could reuse as our defaults.

Reply:
No insight from the Cloud team. Going to leave this as-is for now, we can revisit later.